tools/isobuild/meteor-npm.js

```diff
@@ -1136,7 +1143,8 @@ var minimizeDependencyTree = function (tree) {
   var minimizeModule = function (module) {
     var version;
     if (module.resolved &&
-        !module.resolved.match(/^https?:\/\/registry.npmjs.org\//)) {
+        !module.resolved.match(/^https?:\/\/registry.npmjs.org\//) &&
+        !module.resolved.startsWith(process.env.NPM_CONFIG_REGISTRY)) {
```

Comment:
On second thought, let's guard against the possibility that `process.env.NPM_CONFIG_REGISTRY` is not defined?